Decentralized Voices (#5584)
w3f/polkadot-wiki: docs/general/decentralized-voices.md

---
id: decentralized-voices
title: Decentralized Voices Program
sidebar_label: Decentralized Voices
description: Learn about the Decentralized Voices Program.
keywords: [community, ecosystem, governance, decentralized voices, delegations]
slug: ../decentralized-voices
---

The Decentralized Voices program for {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}
empowers broader community participation in [Polkadot OpenGov](../learn/learn-polkadot-opengov.md)

> by delegating {{ polkadot: 42 Million DOT :polkadot }}{{ kusama: 180,000 KSM :kusama }}. Interested

Reviewer: It is actually 7 Million DOT and 30,000 KSM delegated with 6X conviction. That is why the medium article mentioned "voting power".

```suggestion
by delegating {{ polkadot: 42 Million DOT :polkadot }}{{ kusama: 180,000 KSM :kusama }} of voting power. Interested
```

applicants can fill
[**this application form**](https://docs.google.com/forms/d/e/1FAIpQLSeIFOhMfoqRoeYqr1c_-UeB2a6u0YEsi9CaNEEeUAXtN0oE3w/viewform)
to qualify for receiving
[voting delegations](../learn/learn-polkadot-opengov.md#multirole-delegation) from the Web3
Foundation. For more detailed information, see
[the original Medium article](https://medium.com/web3foundation/decentralized-voices-program-93623c27ae43).

Building on the success of previous initiatives like
[the Thousand Validators Program](./thousand-validators.md) and
[the Decentralized Futures Program](./decentralized-futures.md), the Web3 Foundation emphasizes the

> importance of decentralized governance within the Polkadot ecosystem to ensure a fair

Reviewer: ```suggestion
[the Decentralized Futures Program](./decentralized-futures.md), Web3 Foundation is working on
decentralizing governance within the Polkadot ecosystem to ensure a fair
```

> decision-making process. In line with this ethos, the Foundation seeks to amplify the voices of

Reviewer: ```suggestion
decision-making process. In line with the Foundation's ethos, this program seeks to amplify the voices of
```

passionate and knowledgeable participants who may lack significant
{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} holdings, aiming to mitigate potential
marginalization. This initiative does not impact existing programs like the Thousand Validators
Programme, as

> [staking and governance locks can coexist](../learn/learn-polkadot-opengov.md#voluntary-locking-conviction-voting).

Reviewer: ```suggestion
[the tokens allocated can be used for both staking and governance](../learn/learn-polkadot-opengov.md#voluntary-locking-conviction-voting).
```

The Decentralized Voices program initially designates delegates who will be receiving delegated
funds. Participants are encouraged to showcase their governance contributions and political
philosophy through public declarations and active engagement on relevant platforms such as:

- [The Polkadot Forum](https://forum.polkadot.network/)
- [X (formerly Twitter)](https://twitter.com/)
- [Reddit](https://www.reddit.com/r/Polkadot/)
- [Discord](https://discord.gg/polkadot)
- [Subsocial](https://polkaverse.com/)

Selection criteria include demonstrated participation in governance activities and ethical voting
practices. Delegations are subject to review, and non-compliance may result in revocation.

> The Decentralized Voices program commences on **February 6, 2024**, offering an opportunity for

Reviewer: ```suggestion
The Decentralized Voices program commenced on **February 6, 2024**, offering an opportunity for
```

---
id: decentralized-voices
title: Decentralized Voices Program
sidebar_label: Decentralized Voices
description: Learn about the Decentralized Voices Program.
keywords: [community, ecosystem, governance, decentralized voices, delegations]
slug: ../decentralized-voices
---

The Decentralized Voices program for {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}
empowers broader community participation in [Polkadot OpenGov](../learn/learn-polkadot-opengov.md)
by [delegating](../learn/learn-polkadot-opengov.md#multirole-delegation)
{{ polkadot: 42 Million DOT :polkadot }}{{ kusama: 180,000 KSM :kusama }} of voting power.

> ## Decentralized Voting Power

Reviewer: ```suggestion
## Decentralized Votes
```

  (Element), or email.
- A link to a _publicly accessible_ declaration of your Polkadot “political philosophy” and/or
  agenda on the [Polkadot Forum](https://forum.polkadot.network/). This can include an actual
  philosophical statement, analogies to other political philosophies, opinions on previous Referenda
  or governance decisions, or any other way you would like to explain to people how you would vote
  in the future. This should be at least one paragraph.
- A description of why you would be a good choice for this program. This can include such
  information as previous votes (including votes with other accounts if you can prove you own them),
  as well as commentary, blog posts, etc. on Polkadot OpenGov or specific referenda. It can also
  include other information that you would consider relevant to decision making, e.g. being part of
  other political groups or legislative bodies.

The Decentralized Voices program initially designates delegates who will be receiving delegated
funds. In addition to the platforms mentioned above, to expand their reach, participants can
showcase their governance contributions and political philosophy through public declarations and

> active engagement on other social media platforms such as:

Reviewer: ```suggestion
active engagement (optionally) on other social media platforms such as:
```
